common/lib/xmodule/xmodule/graders.py

```diff
@@ -193,11 +196,19 @@ def grade(self, grade_sheet, generate_random_scores=False):
 
             total_percent += weighted_percent
             section_breakdown += subgrade_result['section_breakdown']
-            grade_breakdown.append({'percent': weighted_percent, 'detail': section_detail, 'category': category})
-
-        return {'percent': total_percent,
-                'section_breakdown': section_breakdown,
-                'grade_breakdown': grade_breakdown}
+            grade_breakdown.append({
+                'percent': weighted_percent,
+                'detail': section_detail,
+                'category': category,
+                'is_passed': subgrade_result['percent'] >= passing_grade
+            })
+
+        return {
+            'percent': total_percent,
+            'section_breakdown': section_breakdown,
+            'grade_breakdown': grade_breakdown,
+            'sections_passed': all(section['is_passed'] for section in grade_breakdown)
+        }
 
 
 class SingleSectionGrader(CourseGrader):
```

Comment:
Do we to add `passing grade` logic in this class?
It looks like it is possible to use it https://github.com/polesye/edx-platform/pull/1/files#diff-743e6bcf9ca89e1c3a85c3ec2c9d81b5R89